tools/analyzer_plugin/lib/src/assist/convert_class_or_function_component.dart

```diff
@@ -17,10 +17,10 @@ const _convertToFunctionDetails = r'''''';
 
 class ConvertClassOrFunctionComponentAssistContributor extends AssistContributorBase {
   @DocsMeta(_convertToClassDesc, details: _convertToClassDetails)
-  static const convertToClass = AssistKind('convertToClass', 30, _convertToClassDesc);
+  static const convertToClass = AssistKind('convertToClass', 40, _convertToClassDesc);
 
   @DocsMeta(_convertToFunctionDesc, details: _convertToFunctionDetails)
-  static const convertToFunction = AssistKind('convertToFunction', 30, _convertToFunctionDesc);
+  static const convertToFunction = AssistKind('convertToFunction', 40, _convertToFunctionDesc);
```

Comment:
I lowered the priority of these (higher number = lower priority) since they were showing up before more commonly-used assists like add ref / add props